app/assets/javascripts/header.js

```diff
@@ -1,11 +1,11 @@
-$(document).ready {
-$(window).scroll(function() {    
-    var scroll = $(window).scrollTop();
-    console.log(scroll)
-    if (scroll >= 500) {
-        $("nav").addClass("scrolling");
-    } else {
-        $("nav").removeClass("scrolling");
-    }
-});
-}
\ No newline at end of file
+$(document).ready(function(){
+	$(window).scroll(function() { 
```

Comment:
Apply the same thing here. In this case, every single time there's a scroll event, we're getting a new reference to `window` and '#nav'. So if you define `$window` and `$nav` outside the scroll callback, it'll save a lot of effort.